tools/pepsi/src/upload.rs

```diff
@@ -127,3 +170,19 @@ pub async fn upload(arguments: Arguments, repository: &Repository) -> anyhow::Re
 
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+
+    fn test() {
+        let input = r#"ID=hulks-os
+    NAME="HULKs-OS"
+    VERSION="5.1.3 (langdale)"
+    VERSION_ID=5.1.3
+    PRETTY_NAME="HULKs-OS 5.1.3 (langdale)"
+    DISTRO_CODENAME="langdale"#;
+
+        let output = extractVersionNum(input);
```

Comment:
Is this function defined somewhere or did I overlook it?

Reply:
Function re-inserted